docs/api-docs/cart-and-checkout/embedded-checkout-tutorial.md

```diff
@@ -163,23 +173,25 @@ Next, generate a cart URL and set this cart as the active cart by posting to  to
 ```
 
 ### Redirecting A Logged-In Customer to Embedded Checkout
-For some use cases, you may want your your customer to be logged in before they can begin the checkout process.
+For some use cases, you may want your customer to be logged in before they can begin the checkout process.
 
-The can be done using the [Customer Login API](https://developer.bigcommerce.com/api-docs/customers/customer-login-api#logging-in-a-customer). 
+This can be done using the [Customer Login API](https://developer.bigcommerce.com/api-docs/customers/customer-login-api#logging-in-a-customer). 
 
-Your app will need to use JSON Web Token Standard to create a new token. Use a [JWT library](https://jwt.io/#libraries-io) to accomplish this. Include the `checkout_url` as part of the request payload you send to BigCommerce:
+Your app will need to use [JSON Web Token Standard](https://jwt.io/introduction/) to create a new token. Use a [JWT library](https://jwt.io/#libraries-io) to accomplish this. Include the `checkout_url` as part of the request payload you send to BigCommerce.
```

Comment:
This paragraph should be broken up into multiple steps.
We don't need to point people to the same resource twice in two sentences.  
Point people to the tutorial we have on how to achieve this: https://developer.bigcommerce.com/api-docs/customers/customer-login-api#create-jwt-using-the-debugger-tool